tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/context/BMainInstance.java

```diff
@@ -413,12 +441,7 @@ private boolean isLeechingDone(ServerLogReader infoReader, ServerLogReader error
      * @param cmdArgs ballerina command args.
      */
     private void terminateProcess(Process process, String[] cmdArgs) {
```

Comment:
`terminateProcess(Process process, String[] cmdArgs)` method can be further simplified using its overloaded method as follows.

```java
private void terminateProcess(Process process, String[] cmdArgs) {
        // Extracts the debug port from the command arguments.
        String port = "";
        for (int i = 0; i < cmdArgs.length; i++) {
            if (cmdArgs[i].equals("--debug")) {
                port = cmdArgs[i + 1];
                terminateProcess(process, port);
                break;
            }
        }
    }
```

Reply:
Fixed